platforms/hyperledger-fabric/configuration/roles/create/peers/tasks/main.yaml

```diff
@@ -21,6 +21,9 @@
     name: "{{ item.name | lower }}"
     type: "value_peer"
     component_name: values-{{ peer.name }}
+    couchdb_image: "hyperledger/fabric-couchdb:{{ couchdb_image_version[fabric_version] }}"
+    peer_image: "hyperledger/fabric-peer:{{ fabric_version }}"
```

Comment:
same here

Reply:
I have used fabric_version as we dont have variable nesting support in jinja like {{ var1{{ var2 }} }}
Hence I am fetching it as a variable from a level above (deploy-network.yaml)